.clang-tidy

```diff
@@ -26,9 +26,18 @@ Checks:     '
             performance-*,
             portability-*,
             readability-*,
+            -bugprone-exception-escape,
```

Comment:
Suggest reenabling in the future as it seems to find a few legit issues with destructors that can throw. Will open an issue.

Reply:
I'll open issues for each one and mark them beginner after this merges. I think that's more visible than comments in a clang-tidy file that no one looks at.